compiler/modules/front/codeGen.bal

```diff
@@ -958,22 +958,15 @@ function codeGenExpr(CodeGenContext cx, bir:BasicBlock bb, Environment env, s:Ex
         }
         // Type test
         var { td, left, semType } => {
-            return codeGenTypeTest(cx, bb, env, td, left, semType);
+            return codeGenTypeTest(cx, bb, env, td, left, semType, false);
+        }
+        //Type not test
+        var { notTd, left, semType } => {
+            return codeGenTypeTest(cx, bb, env, notTd, left, semType, true);
         }
         // Variable reference
         var { varName } => {
-            var v = check lookupVarRef(cx, varName, env);
```

Comment:
Why is this needed for `!is`?

Reply:
moved it to avoid method too large error